charts/celestia-node/templates/statefulset.yaml

```diff
@@ -188,6 +188,10 @@ spec:
           ports:
             - name: p2p
```

Comment:
Q: should we keep the `TCP` port name to `p2p`, or should we be explicit and name it `p2p-tcp`?

Reply:
yeah makes sense to be consistent and have them clearly separated. I've appended `tcp` to the initial ones